assets/src/blocks/Covers/CoversConstants.js

```diff
@@ -12,3 +14,53 @@ export const COVERS_LAYOUTS = {
 export const CAROUSEL_LAYOUT_COVERS_LIMIT = 12;
 
 export const BLOCK_NAME = 'planet4-blocks/covers';
+
+export const COMMON_ATTRIBUTES = {
```

Comment:
This part of the code has been modified since Sonar reported `"Duplicated Lines (%) on New Code
76,3%"`.

Reply:
I find it weird that we have to split these attributes, sometimes we can ignore SonarCloud and I would do that here, what do you think?

Reply:
I agree with @mleray, especially considering that we want to get rid of Cover blocks soon. For me, it does not make sense to introduce many changes to them.

Reply:
Generally speaking, as you say @GP-Dan-Tovbein  we always have to avoid duplicated code. 
However, IMO we should change this block as minimum as possible as we'll eliminate it soon.

Reply:
@mardelnet just cuoriosly, what do you mean about "soon"? Are we closer to that date or is a plan for the future? From my point of view.. if we are no closer to the date, I don't think if making tiny changes will affect it. We might always to keep this and all blocks working stable, even if we're going to remove them afterwards.

But again, it's a point of view! If tou think it's fine to revert that change I'll do it! 
What do you both think?

Reply:
Fair enough asking the meaning of "soon".
The answer is: I don't really know.
I just know that we already have 2 migration tickets ([here](https://github.com/greenpeace/planet4-master-theme/pull/2363) and [here](https://github.com/greenpeace/planet4-master-theme/pull/2349)) for the Cover block, but realistically speaking, completing the migration could take several weeks.